uikit/src/main/java/com/midtrans/sdk/uikit/activities/BaseActivity.java

```diff
@@ -303,8 +311,23 @@ protected void attachBaseContext(Context newBase) {
         super.attachBaseContext(localizationDelegate.attachBaseContext(newBase));
     }
 
+    private MidtransSDK getMidtransSdk() {
+        MidtransSDK midtransSDK = MidtransSDK.getInstance();
+        if (midtransSDK == null || midtransSDK.isSdkNotAvailable()) {
+            onNullInstanceSdk();
+        }
+
+        return midtransSDK;
+    }
+
     private String getLanguageCode() {
-        return MidtransSDK.getInstance().getLanguageCode();
+        String languageCode = LANGUAGE_CODE_EN;
+        try {
+            languageCode = getMidtransSdk().getLanguageCode();
```

Comment:
above we already tried to check for null and not available sdk, do you think it is necessary to also catch an exception here?

Reply:
yes you are correct, was removed please check @theanh0512